src/core/api/public_api.ts

```diff
@@ -611,7 +622,58 @@ class Player extends EventEmitter<IPublicAPIEvent> {
   loadVideo(opts : ILoadVideoOptions) : void {
     const options = parseLoadVideoOptions(opts);
     log.info("API: Calling loadvideo", options);
+    this._priv_lastContentPlaybackInfos = { options };
+    this._priv_initializeContentPlayback(options);
+  }
 
+  /**
+   * Reload last content. Init media playback without fetching again
+   * the manifest.
+   * @param {Object} reloadOpts
+   */
+  reload(reloadOpts?: { startAt?: { position?: number; relative?: number } }): void {
+    const { options,
+            manifest,
+            lastPlaybackPosition } = this._priv_lastContentPlaybackInfos;
+    if (options === undefined ||
+        manifest === undefined ||
+        lastPlaybackPosition === undefined) {
+      throw new Error("API: Can't reload without having previously loaded a content.");
+    }
+    let startAtPositon: number;
+    if (reloadOpts !== undefined &&
+        reloadOpts.startAt !== undefined &&
+        reloadOpts.startAt.position !== undefined) {
```

Comment:
difference with `reloadOpts?.startAt?.position !== undefined`?
In case it is `null`?

Reply:
Didn't handle the `null` case. My bad.